docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -312,42 +312,6 @@ The Java code generated for this variant is:
     public Variant toValue() { /* ... */ }
   }
 
-Parameterized Types
-^^^^^^^^^^^^^^^^^^^
-
-.. note::
-
-   This section is only included for completeness: we don't expect users to make use of the ``fromValue`` and ``toValue`` methods, because they would typically come from a template that doesn't have any unbound type parameters.
-
-The Java codegen uses Java Generic types to represent :ref:`Daml parameterized types <daml-ref-parameterized-types>`.
-
-This Daml fragment defines the parameterized type ``Attribute``, used by the ``BookAttribute`` type for modeling the characteristics of the book:
-
-.. literalinclude:: ./code-snippets/Com/Acme/ParameterizedTypes.daml
-   :language: daml
-   :start-after: -- start snippet: parameterized types example
-   :end-before: -- end snippet: parameterized types example
-   :caption: Com/Acme/ParametrizedTypes.daml
-
-The Java codegen generates a Java file with a generic class for  the ``Attribute a`` data type:
-
-.. code-block:: java
-  :caption: com/acme/parametrizedtypes/Attribute.java
-  :emphasize-lines: 3,8,10
-
-  package com.acme.parametrizedtypes;
```

Comment:
wondering if this is a typo? Should parametrizedtypes be paramet**e**rizedtypes?

Reply:
Yes, fixed with f7cd9ad268404dd7c7d9cfca52a8fa400bd103a4